% Template for PLoS
% Version 1.0 January 2009
%
% To compile to pdf, run:
% latex plos.template
% bibtex plos.template
% latex plos.template
% latex plos.template
% dvipdf plos.template

\documentclass[10pt]{article}


% amsmath package, useful for mathematical formulas
\usepackage[hyphens]{url}
\usepackage{lineno,hyperref}

\usepackage{amsmath}
% amssymb package, useful for mathematical symbols
\usepackage{amssymb}

\usepackage[normalem]{ulem}

% graphicx package, useful for including eps and pdf graphics
% include graphics with the command \includegraphics
\usepackage{graphicx}

% cite package, to clean up citations in the main text. Do not remove.
%% \usepackage{cite}
\usepackage[authoryear]{natbib}

% Use an edited PLoS provided bibtex style
\bibliographystyle{myunsrt}


\usepackage{color} 
\usepackage{float}
\usepackage{multicol}

%% Use doublespacing - comment out for single spacing
\usepackage{setspace} 
%% \doublespacing


% Text layout
\usepackage[margin=1in]{geometry}
%% \topmargin 0.0cm
%% \oddsidemargin 0.5cm
%% \evensidemargin 0.5cm
%% \textwidth 16cm 
%% \textheight 21cm

% Bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,justification=raggedright]{caption}


% Change bib name
\renewcommand\refname{REFERENCES}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Set figure widths
\newcommand{\figwidthonecol}{7.9375cm}
\newcommand{\figwidthoneandhalfcol}{12.8588cm}
\newcommand{\figwidthtwocol}{16.8275cm}


% Leave date blank
\date{}

\pagestyle{myheadings}


%% ** EDIT HERE **

\makeatother

% \usepackage{xunicode}
\begin{document}

%% \begin{spacing}{2}

\section*{SUPPLEMENTAL METHODS}

\subsection*{Ethics statement}

All human research was conducted in accordance with approved IRB protocols.
Patient specimens were collected at MSKCC under IRB protocol \#00-144 A.
All patients provided written informed consent for use of their sample for general research purposes, and did not specify limitations that restricted use of their samples for this study.
Analysis of samples was conducted under NCI Protocol \#13CN024 without obtaining further consent, as patients had provided prior consent and the samples were analyzed anonymously.
This study was performed in strict accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health.
The protocol was approved by the National Cancer Institute's Animal Care and Use Committee (Protocol DB-091).

\subsection*{Tumor samples}
Frozen sections of each tumor sample obtained from the Memorial Sloan Kettering Cancer Center (MSKCC) tumor bank were H\&E stained and reviewed under the microscope for presence of viable tumor.
Diagnosis was confirmed by a board-certified dermatopathologist (K.B.) Only tumor samples with at least 80\% cell purity were used.
All normal samples were also analyzed by H\&E stained sections from the frozen tissue and the absence of tumor was confirmed by microscopic examination by a pathologist (K.B.).

\subsection*{MCV detection by real time quantitative PCR}

Primers were designed to amplify a  region of the Merkel cell polyomavirus T antigen (coordinates 444-579 in the ``MCV350'' sequence GenBank Accession Number EU375803) to test MCV viral status in tumor samples.
Primers for $\beta$-actin (\emph{ACTB}) were used as a reference.
50 ng of DNA were combined with primers (1 ul of 10 uM stock) and SsoFast EvaGreen Supermix (Biorad, Hercules, CA).
Cycling was done as follows: $95^{\circ}\mathrm{C}$ x 30 sec, followed by 40 cycles of $95^{\circ}\mathrm{C}$ x 1 sec, $58.3^{\circ}\mathrm{C}$ for 15 sec.
A melt curve was determined at $65-95^{\circ}\mathrm{C}$, increment $0.5^{\circ}\mathrm{C}/5\mathrm{~seconds}$.
Fluorescence was detected using a BioRad CFX96.
DNA from the MCV positive Mkl-1 cell lines was used as a positive control, and relative quantities of MCV were determined using the $\Delta\Delta\mathrm{C}_t$ method.
Primer sequences are as follows: \emph{ACTB} (reference, ref. \citep{Huang2007Expression}): forward TCACCCACACTGTGCCCATCTACGA and reverse CAGCGGAACCGCTCATTGCCAATGG (expected product length 294bp);
MCV T antigen: MCV444F forward TCCTTGGGAAGAATATGGAACT and MCV579R reverse GCGAGACAACTTACAGCTAA (expected product length 136bp).
Samples were deemed virus positive if the relative MCV copy number was greater than 0.01, and virus negative if the copy number was less than 0.0001 (in comparison to Mkl-1).
See Figure S3.

\subsection*{Cell lines}
The Merkel cell carcinoma cell lines Mkl-1 \citep{Rosen1987Establishment}, WaGa \citep{Houben2010Merkel}, MCC13, MCC26 \citep{Leonard1995Characterisation}, and UISO \citep{Ronan1993Merkel} have been described previously.
SK-MC01 (MC01) was established from an 80 year old male patient with an 8.5 x 6.5 cm infiltrative MCC tumor on the left buttock.
Tumor immunostaining for chromogranin and synaptophysin were positive.
Staining for KRT20, CAM 5.2, and AE1/AE3 were positive with a paranuclear dot-like pattern.
Another fragment of the patient’s tumor was flash frozen and analyzed as MCC tumor sample MT23.

Mkl-1, WaGa, MCC13, MCC26 and UISO cells were grown and passaged in RPMI-1640 medium (Gibco by Life Technologies) supplemented with 10\% FBS (Hyclone – Thermo Fisher), 100 U/mL penicillin, 0.1 mg/mL streptomycin, and 0.1 mg/ml Normocin (InvivoGen).
MC01 cells were grown and passaged in RPMI-1640 medium (Gibco by Life Technologies) supplemented with 10\% FBS (Hyclone – Thermo Fisher), 100 U/mL penicillin, 0.1 mg/mL streptomycin, and 1xNEAA.

Mkl-1, WaGa, and UISO late passage cells were obtained from Dr. J\"{u}rgen Becker.
UISO early passage cells were obtained from Dr. Das Gupta.
MC01 cells were generated at the Memorial Sloan Kettering Cancer Center (MSKCC).
MCC13 and MCC26 were obtained from Dr. Patrick Moore.

\subsection*{cDNA array hybridization of tumor samples}
Tumor specimens were supplied by the Memorial Sloan Kettering Cancer Center (MSKCC) tumor bank, Department of Pathology, tissue procurement service.
Total RNA was extracted from tumor specimens using the RNeasy Mini Kit (Qiagen Cat. No. 74104).
All samples were treated with RNase-free DNase (Qiagen Cat. No. 79254).
RNA quality was checked on an Agilent Bioanalyzer 2100.
RNA (100 ng) was reverse-transcribed and labeled with biotin using the Affymetrix $3^{\prime}$ IVT Express Kit following manufacturer's protocol at the MSKCC core facility.
The samples were hybridized to Human Genome U133A 2.0 Array (Affymetrix) GeneChips.
Posthybridization staining and washing were processed according to manufacturer (Affymetrix) guidelines.
The chips were scanned on Affymetrix GeneChip Scanner 3000 7G.
Data were collected using Affymetrix GeneChip Command Console (AGCC) Software.


\subsection*{cDNA array hybridization of cell lines}

Total RNA was extracted from cell lines using the RNeasy Mini Kit (Qiagen Cat. No. 74104).
All samples were treated with RNase-free DNase (Qiagen Cat. No. 79254).
RNA quality was analyzed with the Agilent Bioanalyzer 2100 using the RNA 6000 Nano Kit (Agilent Technologies).

RNA (100 ng) was reverse-transcribed and labeled with biotin using the Affymetrix $3^{\prime}$ IVT Express Kit following manufacturer's protocol.
Six replicates of each cell line (except for MCC13 and MCC26) were prepared, labeled, and hybridized to the Human Genome U133A 2.0 Array (Affymetrix) at the NCI Center for Cancer Research microarray core facility.
Three replicates of each cell line were also hybridized to the GeneChip Human Genome U133 Plus 2.0 Array (Affymetrix).
The chips were scanned on Affymetrix GeneChip Scanner 3000G.
Data were collected using Affymetrix GeneChip Command Console (AGCC) Software.


\subsection*{Microarray analysis of MCC tumors, SCLC tumors, and MCC cell lines (HG-U133A 2.0 Array data)}

All microarray expression analysis was performed using R, a free software environment for statistical computing and graphics.
The R package ``affy'' \citep{Gautier2004Affyanalysis} was used to read in probe intensity data from CEL-formatted files.
The Robust Multi-Array Average (RMA) procedure was used for background correction, quantile normalization and probe set summarization \citep{Irizarry2003Exploration}.
No batch correction was performed, as no batch effects were detected.

After normalization, the tool ``arrayQualityMetrics'' was used for quality control to ascertain comparability of arrays.
All arrays passed the quality controls for outlier detection by distance between arrays and MA plots.
Sample two of cell line Mkl-1 and sample three of cell line WaGa were flagged as outliers using a boxplot of intensity values (computed as the Kolmogorov-Smirnov statistic between the distribution of intensities for individual arrays versus all arrays pooled together).
However these samples were very close to the threshold of $0.027$ ($\mathrm{Mkl\-1}=0.028,\mathrm{WaGa}=0.028$) and were included in all analyses.
The bottom 40\% of probe sets sorted by variance of expression over all samples were removed using the ``genefilter'' package (13,329 probe sets retained after filtering).

The expression data was used to perform principal components analysis using all probe sets.
The expression data was also used to cluster the samples using the hierarchical clustering method from the ``agnes'' package.
One minus the Spearman correlation was used for the distance metric, and average linkage was applied for merging clusters.
The partitioning around mediods (PAM) clustering method from the ``cluster'' package was applied to the same expression data, with results similar to the hierarchical clustering (data not shown).

\subsection*{Differential gene expression analysis}

The package ``limma'' was used for differential expression analysis to compare MCC cell lines to MCC tumor samples, other MCC cell lines, and between MCC tumor samples grouped by clinical features \citep{Smyth2004Linear,Smyth2005Limma}.
All $p$-values were FDR corrected for multiple hypothesis testing ($q$-values \citep{Benjamini1995Controlling}).
We call probe sets to be differentially expressed if the absolute fold change is greater than or equal to 2 and the $q$-value is less than or equal to 0.05.

\subsection*{Classification of MCC cell lines compared to MCC and SCLC tumors}

A random forest classifier was trained using the microarray expression data for the 23 MCC and 9 SCLC tumor samples using the ``randomForest'' package.
Probe Sets were variance-filtered as previously described.
A forest of 1000 trees was used, and samples were drawn with replacement.
The class sizes were balanced using the minimum size of the two classes (8).
To evaluate the random forest classifier, we examined the out-of-bag classification error, which was 0\%.
The classifier was then applied to the MCC cell lines, and the average class prediction was determined for each of the MCC cell lines (across sample replicates).

\subsection*{Microarray analysis of MCC and CCLE cell lines (HG-U133 Plus 2.0 Array data)}

The R package ``affy'' \citep{Gautier2004Affyanalysis} was used to read in probe intensity data from CEL-formatted files obtained from the NCI Center for Cancer Research microarray core facility for the MCC cell lines and the raw CEL files for CCLE data were downloaded directly from GEO (accession GSE36133).
Robust Multi-Array Average normalization was applied to all samples for background correction, quantile normalization and probe set summarization \citep{McCall2010Frozen}.
Batch effects between the MCC cell lines and the CCLE cell lines were observed by examining the results from the ``arrayQualityMetrics'' package, PCA plots, and using the surrogate variable analysis package ``sva.''
Thus, the expression data was subsequently processed using the ``ComBat'' algorithm, a non-parametric empirical Bayes adjustment model \citep{Johnson2007Adjusting} to reduce the impact of ``expression heterogeneity'' from unknown factors.
All arrays passed quality control examination using the R package ``arrayQualityMetrics'' after batch correction.
The bottom 40\% of probe sets sorted by variance of expression over all samples were removed using the ``genefilter'' package (32,768 probe sets retained after filtering).

\subsection*{Classification of MCC cell lines with CCLE cell lines}

A random forest classifier was trained using the batch-corrected microarray probe set expression data for the WaGa and Mkl-1 cell line samples along with cell line samples from the CCLE dataset using the ``randomForest'' package.
Probe Sets were variance-filtered as previously described.
A forest of 1000 trees was used each with $\sqrt{32768}=181$ randomly selected features.
Samples were drawn with replacement, and class sizes were balanced by using the minimum size of all classes (5).
The random forest was subsequently applied to the variant (MCC13, MCC26, and UISO) samples.
%% A second random forest was trained with the UISO samples as a separate class, and the WaGa and Mkl-1 samples labeled as MCC.
%% See the Supplementary Materials for the analysis code.

\subsection*{STR Profiling}

Genomic DNA was isolated from cell lines using the Puregene Core kit A (Qiagen) and sent to the Johns Hopkins Fragment Analysis Facility for STR profiling.
All cell lines besides MCC13 and MCC26 were profiled using the AmpF$\ell$STR Identifiler PCR Amplification Kit (Applied Biosystems), which profiles 16 STR markers.
MCC13 and MCC26 were profiled using the Promega PowerPlex 18D Kit, which profiles the same 16 markers plus two others (Table S6).
The profiles were also compared against profile databases of known cell line references including ATCC, DSMZ, and JCRB.

\subsection*{Mouse xenografts of UISO and WaGa cell lines}

Five to eight-week-old female athymic nude mice were obtained from the NCI Animal Production Program and housed under specific pathogen-free conditions.
Tumors were induced by subcutaneous injection of $10^7$ WaGa cells, $2\mathrm{x}10^7$ Mkl-1 cells, or $10^6$ UISO cells in 200 $\mu\mathrm{L}$ sterile saline into the posterior lateral flank of the mice.
Tumor tissue was collected after tumors were clinically detectable and were fixed in neutral buffered formalin.

\subsection*{Immunohistochemistry of UISO and WaGa mouse xenograft tumors}

Paraffin-embedded tissue sections (5 $\mu\mathrm{m}$) of fixed xenograft tumors on glass slides were stained on an automated immunostainer (Ventana Medical System, Tucson, AZ, USA), according to the manufacturer's instructions.
The primary antibodies and antigen retrieval methods used are indicated in Table S7.

\subsection*{Spectral karyotyping of the UISO cell line}

UISO cells were harvested after Colcemid (Life Technologies, Grand Island, NY) treatment ($0.025 \mu\textrm{g/mL}$: 1–2 hours).
They were processed by standard cytogenetic methods using hypotonic solution (0.075M potassium chloride) and were fixed in methanol-acetic acid (3:1 dilution).
Metaphase spreads were prepared under optimized humidity conditions with a Thermotron Cytogenetic Drying Chamber (Thermotron, Holland, MI) and allowed to dry for 3-5 days.
Slide denaturing was performed in 70\% formamide/2xSSC at $80^{\circ}\mathrm{C}$ for 2 min, followed by dehydration in 70\%, 90\% and 100\% ethanol.

Chromosome-specific painting probes (Applied Spectral Imaging, Migdal Ha’Emek, Israel) were applied to the slide and hybridized for 72h at $37^{\circ}\mathrm{C}$.
Post-hybridization washes were performed three times in 50\% formamide/2xSSC at $45^{\circ}\mathrm{C}$ and three times in 1xSSC.
The biotinylated probe sequences and the digoxin-labeled probe sequences were detected with the detection probe provided by Applied Spectral Imaging.
Slides were washed in 4xSSC/0.1\% Tween(TM) 20 solution at $45^{\circ}\mathrm{C}$, and counter-stained with 4′,6-diamidino-2-phenylindole (DAPI, Sigma, St. Louis, MO, USA).
Image acquisition was performed through a custom-designed optical filter (SKY v.3; Chroma Technology, Brattleboro, VT) using an SD-301-VDS SpectraCube (Applied Spectral Imaging, Carlsbad, CA) and COOL-1300QS CCD camera (VDS Vosskühler GmbH, Osnabrück, Germany) mounted on a Leica DMRXA microscope (Leica, Wetzlar, Germany).
Applied Spectral Imaging software (Spectral Imaging and SKY View) was used for image acquisition and analysis.
Breakpoints on the SKY-painted chromosomes were determined by comparison with corresponding inverted DAPI banding of the same chromosome.
We report SKY results by using the International System for Human Cytogenetic Nomenclature (ISCN, 2013).

\subsection*{Data availability}
The tumor sample and cell line microarray data discussed in this publication has been deposited in NCBI's Gene Expression Omnibus \citep{Edgar2002Gene} and is accessible through GEO Series accession number GSE50451.
All analysis code, including generation of all figures and supplementary tables and figures, is available in reference \cite{DailyUISOReproducible2014}. %% \url{https://github.com/kdaily/UISO\_code/}.

%% \end{spacing}

\bibliography{uisopub,extra}


\end{document}
